.github/workflows/build_push_img.yml

```diff
@@ -71,11 +100,21 @@ jobs:
     environment: marqo-build-environment
 
     steps:
+      - name: Set input variables
+        id: set-inputs
+        run: |
+          echo "MARQO_REF=${{ inputs.marqo_ref || github.event.inputs.marqo_ref }}" >> $GITHUB_ENV
+          echo "PUSH_TO=${{ inputs.push_to || github.event.inputs.push_to }}" >> $GITHUB_ENV
+          echo "IMAGE_REPO=${{ inputs.image_repo || github.event.inputs.image_repo }}" >> $GITHUB_ENV
+          echo "IMAGE_TAG=${{ inputs.image_tag || github.event.inputs.image_tag }}" >> $GITHUB_ENV
+          echo "DOCKERHUB_USERNAME=${{ inputs.dockerhub_username || github.event.inputs.dockerhub_username }}" >> $GITHUB_ENV
+          echo "DOCKERHUB_PASSWORD=${{ inputs.dockerhub_password || github.event.inputs.dockerhub_password }}" >> $GITHUB_ENV
```

Comment:
This might be dangerous if $GITHUB_ENV can be echoed out.

Reply:
I think this is left here because I just picked the build_push_img.yml action from open_source_CI and put it in this repository. I also think we had plans to move that action to this repository, let me know if we plan to continue using that action as a independent action to build & push images to dockerhub / ECR. If not and if we only plan to use this repo's build_push_img.yml action for just pushing to ECR then we can even remove dockerhub related code altogether, otherwise I will create secrets and use them here

Reply:
After discussion with Yihan, I've removed dockerhub related code entirely from this .yml file